packages/react/src/component.tsx

```diff
@@ -29,11 +31,23 @@ export function Uploader<TRouter extends FileRouter>(
   );
 }
 
-export function generateComponents<TRouter extends FileRouter>() {
+export function generateComponents<TRouter extends FileRouter>(initOpts?: {
+  /**
+   * The URL where you expose your UploadThing router.
+   * @default `/api/uploadthing`
```

Comment:
For future proofing's sake it might be good to require this be an absolute url so we can SSR etc